### DISCUSSION TITLE
MMS Images - Message - iPhone

### DISCUSSION THREAD

**Discussion by Elanchezhiyan-P** [artehe/Netimobiledevice#22] [General]
Hey @artehe , 

Do you have any idea on taking the backup of the MMS along with their images? I have seen the message db and we have attachment location as well (Check the image below). Let me know your thoughts on it.

![image](https://github.com/artehe/Netimobiledevice/assets/29861348/b73f6855-d764-45bf-86ae-11eca0f7a432)

**Comment by artehe** [comment]
I'm not too sure, as I don't think I've ever looked for this. Although if the data isn't in a normal backup it's possible that it gets copied over in an encrypted backup as they contain some extra information you don't normally backup

**Reply by Elanchezhiyan-P** [reply]
Thanks for the information, @artehe
